src/main/java/com/gdschongik/gdsc/domain/member/application/AdminMemberService.java

```diff
@@ -32,9 +32,8 @@ public class AdminMemberService {
     private final ExcelUtil excelUtil;
     private final AdminRecruitmentService adminRecruitmentService;
 
-    public Page<AdminMemberResponse> findAllByRole(
-            MemberQueryOption queryOption, Pageable pageable, MemberRole memberRole) {
-        Page<Member> members = memberRepository.findAllByRole(queryOption, pageable, memberRole);
+    public Page<AdminMemberResponse> findAllByRole(MemberQueryOption queryOption, Pageable pageable) {
```

Comment:
```suggestion
    public Page<AdminMemberResponse> searchMembers(MemberQueryOption queryOption, Pageable pageable) {
```

더이상 역할로 검색하지 않기 때문에 이름을 변경해야 합니다
이름은 예시이고 더 적합한 이름 있으면 편하신대로 정하셔도 돼요